lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1363,15 +1349,16 @@ abstract class DefaultTextEditingStrategy with CompositionAwareMixin implements
     subscriptions.add(DomSubscription(domDocument, 'selectionchange',
             handleChange));
 
-    activeDomElement.addEventListener('beforeinput',
-        createDomEventListener(handleBeforeInput));
+    subscriptions.add(DomSubscription(activeDomElement, 'beforeinput',
+            handleBeforeInput));
 
-    addCompositionEventHandlers(activeDomElement);
+    if (this is! SafariDesktopTextEditingStrategy) {
+      // handleBlur causes Safari to reopen autofill dialogs after autofill,
+      // so we don't attach the listener there.
+      subscriptions.add(DomSubscription(activeDomElement, 'blur', handleBlur));
+    }
```

Comment:
the `handleBlur` function was causing trouble with autocomplete in Safari (refocusing the field). Is this maybe not needed anymore?